src/interface/api_swagger.json

```diff
@@ -1597,7 +1597,7 @@
             "$ref": "#/parameters/FSPIOP-Source"
           },
           {
-            "$ref": "#/parameters/FSPIOP-Destination-Optional"
+            "$ref": "#/parameters/FSPIOP-Destination-Required"
```

Comment:
Did you change the Swagger?

If so we need to update the documentation repo with the changes and also bump the version up.

Is this change not breaking?

Reply:
No @elnyry can verify that this isnt a breaking change and that because there was an issue with put parties endpoints not having FSPIOP-Destination as required and it was needed for the recipient. A note has been added to the README.md